ably/realtime_channel_integration_test.go

```diff
@@ -294,6 +294,47 @@ func TestRealtimeChannel_ShouldSetProvidedReadLimit(t *testing.T) {
 	assert.Equal(t, int64(2048), client.Connection.ReadLimit())
 }
 
+func TestRealtimeChannel_SetsNoLimitIfServerNoLimits(t *testing.T) {
+	// Mock out the dial
+	dial := DialFunc(func(p string, url *url.URL, timeout time.Duration) (ably.Conn, error) {
```

Comment:
We can simplify test using `MessagePipe(in, out)` instead of `DialFunc`. Is it fine if I update the test with the same?

Reply:
Done in [514dd2f](https://github.com/ably/ably-go/pull/631/commits/514dd2f021fe4856cff16594153dfc6d936878e9) :)